pay-api/src/pay_api/services/statement.py

```diff
@@ -248,6 +248,7 @@ def get_invoices_owing_amount(auth_account_id: str):
                 .filter(InvoiceModel.invoice_status_code.in_((InvoiceStatus.SETTLEMENT_SCHEDULED.value,
```

Comment:
Do we need an EFT filter here?

Reply:
I think we do

Reply:
yes, let's do that to make this more clean/safe. Only EFT uses the statement summary right now. We can consider expanding this by adding a payment method query parameter in the future if needed.